docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -397,7 +522,80 @@ Furthermore, the REST API consumers associated with Acme applications will be co
 }
 ```
 
-#### 3.4.2.3 Datastore configuration reference
+##### 3.4.2.3.3 Example 3
+
+Example 3 implements the same configuration as Example 2 using standard Liberty datasource definitions.
+
+```
+{
+    "__comment":"Acme's FHIR server configuration",
+    "fhirServer":{
+        …
+        "persistence":{
+            "factoryClassname":"com.ibm.fhir.persistence.jdbc.FHIRPersistenceJDBCFactory",
+            "jdbc": {
+                "enableProxyDatasource": false
+            },
+            …
+            "datasources": {
+                "study1": {
+                    "tenantKey": "<the-base64-tenant-key>",
+                    "type": "db2"
+                },
+                "study2": {
+                    "tenantKey": "<the-base64-tenant-key>",
+                    "type": "db2"
+                }
+            }
+            …
+        }
+    }
+}
+```
+
+Note that because this example is using the default FHIR server naming scheme for datasource JNDI names, there is no need to include the 'jndiName' property, although you can specify it should you wish to make the mapping clear. The "type" must match the actual database type referenced by the datasource definition. If the type does not match, the behavior is undefined.
```

Comment:
```suggestion
Note that because this example is using the default IBM 
FHIR Server naming scheme for datasource JNDI names, there is no need to include the 'jndiName' property, although you can specify it should you wish to make the mapping clear. The "type" must match the actual database type referenced by the datasource definition. If the type does not match, the behavior is undefined.
```

Reply:
ok